src/main/java/com/intern/bookstore/controllers/BookRestController.java

```diff
@@ -0,0 +1,54 @@
+package com.intern.bookstore.controllers;
+
+import com.intern.bookstore.models.Book;
+import com.intern.bookstore.services.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+public class BookRestController {
+
+    @Autowired
+    private BookService bookService;
+
+    @GetMapping("/books")
```

Comment:
```suggestion
    @GetMapping("")
```

Tương tự cho mấy cái sau